rakelib/vet360.rake

```diff
@@ -180,10 +153,9 @@ namespace :vet360 do
 
     body = JSON.parse(ENV.fetch(ENV_VAR_NAME, nil))
     vet360_id = body['vet360_id']
-    ensure_var('vet360_id', vet360_id)
```

Comment:
ensure_var('vet360_id', vet360_id) is no longer needed for ContactInformation requests. ContactInformationV2 does not require vet360_id for updating contact info.